README.md

```diff
@@ -6,6 +6,15 @@
 * `docker-compose run web bundle exec rails db:create db:setup`
 * `docker-compose up`
 
+**Setup Using Visual Studio Code**
```

Comment:
If this is optional, I'd like to label it that way, especially if it only works if you have access to the service. If it totally works as a local solution, then that can be made more clear, too. But if the project still works normally using just Docker, I wouldn't want a candidate to feel like they have to do this part. I just don't know how common knowledge this is so people would be able to decide for themselves whether they should attempt this or not.

Reply:
Luckily doing these steps only requires Docker and VS Code. The feature is already built-in and using the existing container development workflow to make it even easier. But identifying this is an optional route is a good call